frontend/src/scenes/session-recordings/filters/AdvancedSessionRecordingsFilters.tsx

```diff
@@ -0,0 +1,193 @@
+import { ActionFilter } from 'scenes/insights/filters/ActionFilter/ActionFilter'
+import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
+import { PropertyFilters } from 'lib/components/PropertyFilters/PropertyFilters'
+
+import { MathAvailability } from 'scenes/insights/filters/ActionFilter/ActionFilterRow/ActionFilterRow'
+import { LemonLabel } from 'lib/lemon-ui/LemonLabel/LemonLabel'
+import { EntityTypes, FilterType, FilterableLogLevel, RecordingDurationFilter, RecordingFilters } from '~/types'
+import { DateFilter } from 'lib/components/DateFilter/DateFilter'
+import { DurationFilter } from './DurationFilter'
+import { LemonButtonWithDropdown, LemonCheckbox } from '@posthog/lemon-ui'
+import { TestAccountFilter } from 'scenes/insights/filters/TestAccountFilter'
+import { FlaggedFeature } from 'lib/components/FlaggedFeature'
+import { FEATURE_FLAGS } from 'lib/constants'
+
+export const AdvancedSessionRecordingsFilters = ({
```

Comment:
I'll assume this is all moved unchanged and can be ignored 🙈

Reply:
Yep! Just pulled it into a component